src/transformers/trainer.py

```diff
@@ -210,7 +197,7 @@ def __init__(
             )
         set_seed(self.args.seed)
         # Create output directory if needed
-        if self.is_world_master():
+        if self.is_world_main_process():
```

Comment:
maybe we could name it `process_zero` rather than main, is this more explicit?

```suggestion
        if self.is_world_process_zero():
```